VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -66,11 +66,23 @@ def __init__(self,patching, hutil, logger,para_parser):
             self.logger.log(error_msg, True ,'Error')
             self.output_lsblk = {}
         self.devicesToInclude = [] #partitions to be included
-        self.isAnyDiskExcluded = para_parser.includedDisks[CommonVariables.isAnyDiskExcluded]
-        self.includedLunList = para_parser.includeLunList
         self.device_mount_points = []
+        self.isAnyDiskExcluded = False
+        try:
+            if(para_parser.includedDisks != None and para_parser.includedDisks != ''):
+                self.isAnyDiskExcluded = para_parser.includedDisks[CommonVariables.isAnyDiskExcluded]
+                self.logger.log("isAnyDiskExcluded {0}".format(self.isAnyDiskExcluded))
+        except Exception as e:
+             error_msg = "Failed to serialize includedDisks because of error %s , stack trace: %s" % (str(e), traceback.format_exc())
+             self.logger.log(error_msg, True ,'Error')    
+        try:
+            if( para_parser.includeLunList != None and para_parser.includeLunList != ''):
+                self.includedLunList = para_parser.includeLunList
```

Comment:
I don't think this block requires to be wrapped in a try-except block as we are checking None and Empty string.